src/components/AddContactForm.js

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { REGEX } from '../constants/constants';
+import { validateInput } from '../helpers/helper';
+import './AddContactForm.css';
+
+class AddContactForm extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      firstName: '',
+      lastName: '',
+      imageUrl: '',
+      firstNameError: false,
+      lastNameError: false,
+      imageUrlError: false
+    };
+  }
+
+  onFirstNameInputChange = (event) => {
+    const { target } = event;
+    const newFirstName = target.value.trim();
+    this.setState({ firstName: newFirstName }, () => {
+      const { firstName } = this.state;
+      this.setState({ firstNameError: !validateInput(firstName, REGEX.NAME) });
+    });
+  }
+
+  onLastNameInputChange = (event) => {
+    const { target } = event;
+    const newLastName = target.value.trim();
+    this.setState({ lastName: newLastName }, () => {
+      const { lastName } = this.state;
+      this.setState({ lastNameError: !validateInput(lastName, REGEX.NAME) });
+    });
+  }
+
+  onImageUrlInputChange = (event) => {
+    const { target } = event;
+    const newImageUrl = target.value.trim();
+    this.setState({ imageUrl: newImageUrl }, () => {
+      const { imageUrl } = this.state;
+      this.setState({ imageUrlError: !validateInput(imageUrl, REGEX.URL) });
+    });
+  }
+
+  isButtonDisabled = () => {
+    const {
+      firstName,
+      lastName,
+      imageUrl,
+      firstNameError,
+      lastNameError,
+      imageUrlError
+    } = this.state;
+    return !(firstName && lastName && imageUrl) || firstNameError || lastNameError || imageUrlError;
+  }
+
+  handleSubmit = () => {
+    const { firstName, lastName, imageUrl } = this.state;
+    const { onSubmitAction } = this.props;
+    onSubmitAction(firstName, lastName, imageUrl);
+    this.setState({ firstName: '', lastName: '', imageUrl: '' });
+  }
+
+  renderErrorMessage() {
+    const {
+      firstNameError,
+      lastNameError,
+      imageUrlError
+    } = this.state;
+    if (firstNameError || lastNameError || imageUrlError) {
+      return (
+        <div className="ui error message">
+          <div className="header">We had some issues</div>
+          <ul className="list">
+            {firstNameError && <li>Your FIRST NAME has special characters.</li>}
+            {lastNameError && <li>Your LAST NAME has special characters.</li>}
+            {imageUrlError && <li>Your IMAGE is not an url.</li>}
+          </ul>
+        </div>
+      );
+    }
+    return null;
+  }
+
+  render() {
+    const {
+      firstName,
+      lastName,
+      imageUrl,
+      firstNameError,
+      lastNameError,
+      imageUrlError
+    } = this.state;
+    return (
+      <div className="ui column grid">
+        <div className="thirteen wide column">
+          <div className="ui equal width error form">
+            <h1 className="ui header center aligned">Add Contact</h1>
```

Comment:
please move this to constants

Reply:
and basically every string that is not a class name, id or smth like this, should be moved to constants. please do that